src/main.rs

```diff
@@ -105,7 +105,7 @@ fn main() {
 
     // Check if $CARGO_HOME is set before capturing the config environment 
     // if it is, remove it, we build the project in a temporary directory
-    match std::env::var("CARGO_HOME") {
+    if std::env::var_os("CARGO_HOME").is_some() {
         Ok(_) => {
```

Comment:
```suggestion
```

Reply:
Fixed manually, github messed up the line wrapping I think?